integrationtests/run.sh

```diff
@@ -26,6 +26,8 @@ echo "Running evaluator integration tests"
 python $SCRIPT_DIR/evaluator/integrationtest_evaluator.py
 echo "Running supervisor integration tests"
 python $SCRIPT_DIR/supervisor/integrationtest_supervisor.py
+echo "Running metadata processor integration tests"
+python $SCRIPT_DIR/metadata_processor/integrationtest_metadata_processor.py
```

Comment:
missed integraion test somehow

Reply:
This ass on purpose. The metadata processor is not used ATM. We'd need something like this when we do something like TracIn embeddings, but right now the metadata processor is somewhat on ice, which is why we don't test it. If this runs through, we can have it but if it ever causes problems and we're not using the processor I would comment it out again :D

Reply:
thanks for the context!

Reply:
I found it's not immediately runnable. Then I will just comment it!